data/projects/trusted-setups.ts

```diff
@@ -9,14 +9,17 @@ const content: ProjectContent = {
 
 export const trustedSetups: ProjectInterface = {
   id: "trusted-setups",
-  section: "pse",
-  projectStatus: ProjectStatus.ACTIVE,
+  section: "archived",
+  projectStatus: ProjectStatus.INACTIVE,
   content,
   image: "trusted-setups.svg",
   name: "Trusted Setups",
   links: {
     github: "https://github.com/zkparty",
   },
+  extraLinks: {
+    learn: [{ label: "Trusted Setup/P0tion LTS Announcement", url: "TODO" }],
+  },
```

Comment:
@NicoSerranoP is preparing something we will publish to communicate this.
So to be update later